src/presenter/films-presenter.js

```diff
@@ -1,102 +1,150 @@
 import {render, remove} from '../framework/render.js';
-import {SHOW_FILM_COUNT_STEP} from '../const.js';
+import {SHOW_FILM_COUNT_STEP, SortType, UpdateType, UserAction, FilterType} from '../const.js';
 import FilmSectionView from '../view/film-section-view.js';
 import FilmContainerView from '../view/film-container-view.js';
 import LoadMoreButtonView from '../view/load-more-button-view.js';
 import NoFilmCardView from '../view/no-film-card-view.js';
 import SortView from '../view/sort-view.js';
 import FilmPresenter from './film-presenter.js';
-import {updateItem} from '../utils/common.js';
 import {sortFilmsByRating, sortFilmsByDate} from '../utils/task.js';
-import {SortType} from '../const.js';
+import {filter} from '../utils/filter.js';
 
 export default class FilmsPresenter {
 
   #filmSection = new FilmSectionView;
   #filmContainer = new FilmContainerView;
-  #loadMoreButtonComponent = new LoadMoreButtonView();
-  #noFilmComponent = new NoFilmCardView();
-  #sortComponent = new SortView();
+  #loadMoreButtonComponent = null;
+  #noFilmComponent = null;
+  #sortComponent = null;
 
```

Comment:
Добавим поле, где будем хранить информацию о презентере, с открытым модальным окном, если такой появится
```suggestion
#openedFilmPresenter = null;
```